apps/web/.env.example

```diff
@@ -1,3 +1,11 @@
+GITHUB_ID=""
+GITHUB_SECRET=""
+GOOGLE_CLIENT_ID=""
```

Comment:
Can u explain these changeS?

Reply:
please check my other commit in this PR, this one i committed bymistake

Reply:
You'll have to remove that commit ser
Closing the PR for now, please re-open after the commits are fixed